### ISSUE TITLE
Shortcut ctrl+shift+A not working

### ISSUE THREAD

**Issue by V-GAO** [JabRef/jabref#7439]
JabRef 5.2--2020-12-24--6a2a512
Windows 10 10.0 amd64 
Java 14.0.2

The shortcut of adding a new article, ctrl+shift+A, seems not working on my computer. Would you please see to it? What should I do with the issue?

**Comment by tmrd993** [comment]
Did you create a library before using the shortcut? I think it only works if you have a library tab open.

**Comment by V-GAO** [comment]
Yes I did. Not a total amateur. But a few minutes ago, it suddenly worked. Not sure what happened.

**Comment by Siedlerchr** [comment]
Maybe a focus problem?

**Comment by koppor** [comment]
We removed ALL creation shortcuts in the last version. Maybe, we should re-add the existing ones and just remove the one and only conflicting one.

See https://github.com/JabRef/jabref/pull/6960#issuecomment-700630352 and https://github.com/JabRef/jabref/pull/6960#issuecomment-711428679

Maybe this issue is a duplicate of https://github.com/JabRef/jabref/issues/7346
